cmd/keymaster/main.go

```diff
@@ -66,6 +66,7 @@ var (
 	cliUsername    = flag.String("username", "", "username for keymaster")
 	duration       = flag.String("duration", "16h", "Duration of the requested certificates in golang duration format (ex: 30s, 5m, 12h)")
 	checkDevices   = flag.Bool("checkDevices", false, "CheckU2F devices in your system")
+	noU2F          = flag.Bool("noU2F", false, "Don't use U2F as second factor")
```

Comment:
How about adding a -noVipAccess option too?

Reply:
Agreed... let me fold the change in this PR (will do later in the afternoon)